examples/aes/src/aes.rs

```diff
@@ -247,10 +233,10 @@ pub fn aes128_encrypt_block(
         KEY_SCHEDULE_LENGTH,
         KEY_LENGTH,
         ITERATIONS,
-    )
+    ).unwrap()
```

Comment:
Why can't this fail ?

Reply:
The only failure case here is in line 172 (`i < (4 * (nr + 1))`), which shouldn't happen with these parameters. It would be great if we'd have a way to tell this to the hacspec compiler (range types here #93).

Reply:
The hacspec compiler will not be able to prove those kind of arithmetic properties, only the F* backend will. But yeah we should do #93.